src/classes/frDataMigration.cls

```diff
@@ -37,8 +37,8 @@
 * CREATED: 2020 Funraise Inc - https://funraise.io
 * AUTHOR: Alex Molina
 */
-public class frDataMigration implements InstallHandler {
-    public void onInstall(InstallContext context) {
+public class frDataMigration {
```

Comment:
It's basically our holder class for any evolutions.  It was once populated with some migrations, but we deployed that and had to remove the migration code after it ran because it referenced a field that we wanted to delete (which was the reason for the migration).  It would take us some effort to remove the class from the managed package, so I figure we can keep it around in case we have to do any data migrations going forward.  

Alternatively, we could also just have frPostInstall handle that now since that is what calls this